api/tests/unit/domain/models/UserLogin_test.js

```diff
@@ -154,16 +154,16 @@ describe('Unit | Domain | Models | UserLogin', function () {
   });
 
   describe('#isUserBlocked', function () {
-    context('when user reaches the limit failure count', function () {
-      it('returns true', function () {
+    context('when user reaches the limit failure count but is not yet blocked', function () {
+      it('returns false', function () {
         // given
         const userLogin = new UserLogin({ failureCount: 50 });
```

Comment:
Suggestion; on test le  blockedAt dans `return !!this.blockedAt;`, 
le faire apparaître de manière explicite dans le test pour notre moi futur ^^ ?
`const userLogin = new UserLogin({ failureCount: 50, blockedAt: null });`